.cargo/config

```diff
@@ -1,8 +1,10 @@
 [alias]
 # Automatically generates the ast and syntax kinds files
 gen-syntax   = "run --package tools --bin tools -- gen-syntax"
-# Extracts the tests from 
+# Extracts the tests from
 gen-tests    = "run --package tools --bin tools -- gen-tests"
+# Installs ra-lsp-server
+install-lsp = "run --package tools --bin tools -- install-lsp"
```

Comment:
I think this could be just `install --package ra_lsp_server --force`, without code changes?